tests/tools/utils.py

```diff
@@ -118,11 +118,13 @@ def _chunk_graph(
     metadata["graph_name"] = name
     metadata["node_type"] = g.ntypes
 
+    # add node_type_counts
+    metadata["num_nodes_per_type"] = [g.num_nodes(ntype) for ntype in g.ntypes]
```

Comment:
pls update the doc page as well: https://github.com/dmlc/dgl/blob/master/docs/source/guide/distributed-preprocessing.rst#example-mag240m-lsc

Reply:
Will update the documents page as soon as I know the process and who is the POC for updating the webpage.